lib/api/controllers/security.js

```diff
@@ -1442,6 +1479,12 @@ function mDelete (kuzzle, type, request) {
           errorsManager.get('services', 'storage', 'incomplete_delete', errors));
       }
 
+      if (ids.length > 1000) {
+        kuzzle.log.info(`[SECURITY] User "${request.context.user}" deleted the following ${type}s ${ids.join(', ')}... (${ids.length - 1000} more users deleted)."`);
```

Comment:
As it is, all the ids will be displayed in the log line.
Try this:
```suggestion
        kuzzle.log.info(`[SECURITY] User "${request.context.user}" deleted the following ${type}s ${ids.slice(0, 999).join(', ')}... (${ids.length - 1000} more users deleted)."`);
```